presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -596,7 +615,8 @@ private ObjectMetadata getS3ObjectMetadata(Path path, String bucketName, String
                     .run("getS3ObjectMetadata", () -> {
                         try {
                             STATS.newMetadataCall();
-                            return s3.getObjectMetadata(bucketName, key);
+                            GetObjectMetadataRequest getObjectMetadataRequest = new GetObjectMetadataRequest(bucketName, key).withRequesterPays(requesterPaysEnabled);
```

Comment:
please add a new line before `.with...` for readability.
i'd also inline `the variable`:

```
return s3.getObjectMetadata(new GetObjectMetadataRequest(bucketName, key)
    .withRequesterPays(requesterPaysEnabled));
```